lib/db/sprint.ts

```diff
@@ -0,0 +1,96 @@
+import prisma from "./prisma";
+import { Sprint } from "@prisma/client";
+
+/**
+ * Retrieves a single sprint based on provided parameters.
+ *
+ * @export
+ * @async
+ * @param {Sprint} params
+ * @returns {Promise<Sprint | null>} The retrieved sprint object or null
+ */
+export async function getSprint(
+  params: Partial<Sprint>
+): Promise<Sprint | null> {
```

Comment:
Because you're including tasks, you'll need to set the return type equal to the custom `Sprint` type you created in `types.ts` here (otherwise the type safety will prevent us from accessing `sprint.task.property`)

Reply:
Need custom Sprint type here!

Reply:
Got it, thanks!